### PR TITLE
Feature/directory cleanup

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: lambdas/code_exec/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,18 @@
+
+FROM public.ecr.aws/lambda/nodejs:12
+
+
+FROM public.ecr.aws/bitnami/golang:latest
```
Comment (stripedpajamas) [inline lambdas/code_exec/Dockerfile:5]: I think this should be one specifically for lambdas and it should be the base image. Then put a generic Node one on top.

File: models/models.go [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+	RunCommand     string `json:"runCmd"`
+	CompileCommand string `json:"compileCmd"`
+}
+
+// LanguageConfig represents the model matching the languages.json file
+type LanguageConfig map[string]LanguageProperties
+
+// ParseLanguageConfig parses the languages file into a LanguageConfig model
+func ParseLanguageConfig() (LanguageConfig, error) {
+	data, err := ioutil.ReadFile("../../languages.json")
```
Comment (stripedpajamas) [inline models/models.go]: will be cooler if the path is a param to this parse func

File: lambdas/run/main.go [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+
+func parseText(text string) models.CodeProcessRequest {
+	text = strings.Trim(text, " ")
+	chars := []rune(text)
+	textLen := len(chars)
+
+	for idx, c := range text {
+		if c == ' ' {
+			return models.CodeProcessRequest{
+				Code:     string(chars[idx+1 : textLen]),
```
Comment (stripedpajamas) [inline lambdas/run/main.go]: can `text` be indexed/sliced directly? I thought strings already were rune slices

**Review by stripedpajamas — APPROVED** [review]
🥰

File: lambdas/run/main.go (hunk 3 above)
Comment (ghost) [reply lambdas/run/main.go]: updated

File: models/models.go (hunk 2 above)
Comment (ghost) [reply models/models.go]: updated

File: lambdas/code_exec/Dockerfile (hunk 1 above)
Comment (ghost) [reply lambdas/code_exec/Dockerfile:5]: changing in separate pr

**Pull request merged**